### PR TITLE
Add spawining->spawning

### PR DESCRIPTION
**Pull request by cfi-gb**
See https://github.com/search?q=spawining&type=code (Note that https://grep.app/search?q=spawining doesn't have that much hits)

Not sure about other variants like `re(-)?spawining`, please let me know if i should add them.

### CODE REVIEW

**Comment by peternewman** [comment]
> Not sure about other variants like `re(-)?spawining`, please let me know if i should add them.

It certainly won't hurt to have them in, the more the merrier!

**Review by peternewman — COMMENTED** [review]
My comment should have been a comment review really.

**Comment by cfi-gb** [comment]
> My comment should have been a comment review really.

Thanks a lot for the review. I have added a few variants, hope these are correct.

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -22684,6 +22684,8 @@ re-negoziations->re-negotiations
 re-negoziator->re-negotiator
 re-negoziators->re-negotiators
 re-realease->re-release
+re-spawining->respawning
+re-spawning->respawning
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:22688]: It looks like re-spawning is valid too, so perhaps suggest both.

Comment (cfi-gb) [reply codespell_lib/data/dictionary.txt]: Ah, indeed. Updated in ea13c9205bf06e0682bf0298bff80a9b8c8d6a49, thanks a lot.

**Review by peternewman — APPROVED** [review]

**Pull request merged**